src/core/catalog.js

```diff
@@ -638,10 +639,39 @@ class Catalog {
       return { name: stringToPDFString(nestedName), order: nestedOrder };
     }
 
+    function parseRBGroups(rbgrps) {
+      if (!Array.isArray(rbgrps)) {
+        return null;
+      }
+
+      // iterate over RB groups (literal arrays or refs thereof)
+      for (const value of rbgrps) {
+        const rbGroup = value instanceof Ref ? xref.fetchIfRef(value) : value;
```

Comment:
This is actually redundant, see [here](https://github.com/mozilla/pdf.js/blob/35a9a6a7b34030397c2215ebd3d17064ac53887e/src/core/xref.js#L794-L799), and either of the following options are correct:
```suggestion
        const rbGroup = value instanceof Ref ? xref.fetch(value) : value;
```
or

```suggestion
        const rbGroup = xref.fetchIfRef(value);
```

Reply:
I will use the second suggestion.